src/java/io/bazel/rulesscala/scalac/ScalaCInvoker.java

```diff
@@ -314,17 +314,42 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
       });
     }
   }
-  private static void copyResources(Map<String, String> resources, Path dest) throws IOException {
+  private static void copyResources(
+      Map<String, String> resources,
+      String resourceStripPrefix,
+      Path dest) throws IOException {
     for(Entry<String, String> e : resources.entrySet()) {
       Path source = Paths.get(e.getKey());
       String dstr = e.getValue();
       if (dstr.charAt(0) == '/') dstr = dstr.substring(1);
+      // Check if we need to modify resource destination path
+      if (!"".equals(resourceStripPrefix)) {
+        dstr = getResourcePath(source, resourceStripPrefix);
+      }
       Path target = dest.resolve(dstr);
       File tfile = target.getParent().toFile();
       tfile.mkdirs();
       Files.copy(source, target);
     }
   }
+  private static String getResourcePath(
+      Path source,
+      String resourceStripPrefix) throws RuntimeException {
+    String sourcePath = source.toString();
+    // check if the Resource file is under the specified prefix to strip
+    if (!sourcePath.startsWith(resourceStripPrefix)) {
+      // Resource File is not under the specified prefix to strip
+      throw new RuntimeException("Resource File is not under the specified prefix to strip");
+    }
+    /**
+     * Using a simple String replaceFirst for now,
+     * Once we get merge into Bazel, we can use the PathFragments from
```

Comment:
what is this comment about? There is no plan, that I know of, to put this back in bazel. This rule (as many rules) are intentionally developed outside so that it can evolve more quickly. As a note, it used to be in bazel mainline source.